core-bundle/src/Doctrine/Backup/BackupManager.php

```diff
@@ -72,20 +82,17 @@ public function getLatestBackup(): ?Backup
      */
     public function listBackups(): array
     {
-        // Ensure that the directory exists
-        (new Filesystem())->mkdir($this->backupDir);
-
-        $files = Finder::create()
-            ->files()
-            ->in($this->backupDir)
-            ->depth('== 0')
-            ->name(Backup::VALID_BACKUP_NAME_REGEX)
-        ;
-
         $backups = [];
 
-        foreach ($files as $file) {
-            $backups[] = new Backup($file->getPathname());
+        foreach ($this->vfs->listContents('', false, VirtualFilesystemInterface::BYPASS_DBAFS) as $file) {
+            try {
```

Comment:
We should probably ignore directories when listing:

```suggestion
            if (!$file->isFile()) {
                continue;
            }

            try {
```

Reply:
Sounds like a great addition to the VFS itself? A flag I mean? Wdyt?

Reply:
You mean for only listing files? Sounds good to me.